strawberryfields/backends/base.py

```diff
@@ -478,6 +479,24 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
         Returns:
             float: measured value
         """
+        print("HI")
+        raise NotImplementedError
+          
+    
+    def measure_fock(self, modes, shots=1, select=None, **kwargs):
+        """Measure the given modes in the Fock basis.
+
+        Updates the current state of the circuit to the conditional state of this measurement result.
```

Comment:
The phrase "circuit state" or "state of the circuit" appears in various places in the code. I think this is a misnomer that should be changed to "system state" or "register state", since circuit refers to a sequence of quantum operations, which corresponds to a transformation acting on the system.

Reply:
Maybe "register state"?

Reply:
Updating here, but otherwise seems out of scope for the PR. We can make a note to do this in a future polishing stage